python/variable.cpp

```diff
@@ -80,12 +80,53 @@ template <class ST> struct MakeODFromNativePythonTypes {
   }
 };
 
+template <class T>
+Variable bind_init_1D_no_variance(const std::vector<Dim> &labels,
```

Comment:
Misleading function name, if is not binding anything (not calling pybind's `def`)?